### PR TITLE
Pull in handwritten examples with new linting

### PR DESCRIPTION
**Pull request by timothymcmackin**

### CODE REVIEW

File: openapi.json [hunk 1]
```diff
@@ -1509,7 +1509,7 @@
           },
           {
             "lang": "javascript--nodejs",
-            "source": "const sstk = require('shutterstock-api');\r\n\r\nsstk.setAccessToken(process.env.SHUTTERSTOCK_API_TOKEN);\r\n\r\nconst imagesApi = new sstk.ImagesApi();\r\n\r\nconst body = {\r\n  \"name\": \"My collection\"\r\n};\r\n\r\nimagesApi.createLightbox(body)\r\n.then(function() {\r\n  console.log(\"Success!\");\r\n})\r\n.catch(function(error) {\r\n  console.error(error);\r\n});\r\n"
+            "source": "const sstk = require('shutterstock-api');\r\n\r\nsstk.setAccessToken(process.env.SHUTTERSTOCK_API_TOKEN);\r\n\r\nconst imagesApi = new sstk.ImagesApi();\r\n\r\nconst body = {\r\n  \"name\": \"My collection\"\r\n};\r\n\r\nimagesApi.createLightbox(body)\r\n  .catch((error) => {\r\n    console.error(error);\r\n  });\r\n"
```
Comment (msmol) [inline openapi.json:1512]: what happened to the `.then` block here?

Comment (timothymcmackin) [reply openapi.json:1512]: I took the `then` block out of methods that don't return any data. Should I put it back in and write "Done!" to the console or something like that?

Comment (msmol) [reply openapi.json:1512]: I think that may make sense. @anthonyjonahs what do you think?

**Review by msmol — APPROVED** [review]

File: openapi.json (hunk 1 above)
Comment (anthonyjonahs) [reply openapi.json:1512]: I think it's fine for a `POST` request like this.

**Review by anthonyjonahs — APPROVED** [review]

**Pull request merged**